Tests/SwiftGenTests/ConfigReadTests.swift

```diff
@@ -44,6 +44,32 @@ class ConfigReadTests: XCTestCase {
     }
   }
 
+  func testConfigWithENVS() throws {
+    guard let paramsConfigFilePath = Bundle(for: type(of: self)).path(forResource: "config-with-params", ofType: "yml") else {
+      fatalError("Fixture not found")
+    }
+
+    // this test needs the env varibale for "SWIFTGEN_OUTPUT_DIR" set to "Common/Generated" in the test section of the swiftgen scheme
+    guard let envConfigFilePath = Bundle(for: type(of: self)).path(
+      forResource: "config-with-env_placeholder",
```

Comment:
This filename breaks a bit with the other files, we probably want "config-with-env-placeholder" here.